kbase/narrative
# DATAUP-330 merge develop -> truss

# Description of PR purpose/changes

This branch combines and normalizes all the code formatting and test changes in the develop branch with the concurrent changes to the truss branch, so we can migrate from develop -> truss.

There are still a number of tests on the truss side that aren't working, all involving the bulk import cell and how its various input widgets get managed. These all seem to have to do with now the bus processes and sends messages asynchronously, often after tests complete. They're skipped for now to ensure that everything else is stable, I'd recommend coming back to those at a later date.

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-330

# Testing Instructions
* Details for how to test the PR:
* Tests should pass
* The bundle should build and deploy on narrative-refactor
* Should be no other visible changes

# Dev Checklist:
